### PR TITLE
[Add] Project Idea: Publishing Posts & Customized Themes

### PR DESCRIPTION
**Pull request by SiddheshKukade**
**What kind of change does this PR introduce?**
Docs
**Issue Number:**
No issue linked

**Did you add tests for your changes?**
Not required
**Snapshots/Videos:**
 
image

**If relevant, did you update the documentation?**
Yes 
**Does this PR introduce a breaking change?**
 No
**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: docs/internships/ideas.md [hunk 1]
```diff
@@ -71,33 +71,33 @@ Most of our project ideas require knowledge of two or more programming languages
...
-            1. The plugin must provide a comprehensive set of advertising campaign features.
-            1. Integration with other advertising platforms should be considered.
-    2. **Spam Mitigation**:
-        1. Detect whether a a user is spamming organization chat and news feed messages.
-        2. **Expected Outcomes:**
-            1. A reliable method to intercept new messages
-            1. An accurate subsystem to evaluate the `spaminess` of messages using both implied and explicit triggers.
-            1. Notification of the Talawa-Admin users of potential spam
-            1. Creation of ways to ban both infringing and related users.
-            1. This should ideally be a plugin so that administrators can eventually select the anti-SPAM engine of their choice.
```
Comment (SiddheshKukade) [inline docs/internships/ideas.md:90]: Some Extra Spaces are removed by code formatted - Prettier.io-

**Comment by SiddheshKukade** [comment]
@palisadoes Sir I would like to have your opinions on this idea and also the `Project Hours` and `Possible Mentors` .

**Comment by palisadoes** [comment]
> @palisadoes Sir I would like to have your opinions on this idea and also the `Project Hours` and `Possible Mentors` fields. Please suggest improvements if required.

1. Please make the mentors `TBD` for now.
1. We are only doing 350 hour projects next year. If you think that the work described does not meet this standard, then please think of another possible plugin.
1. Please mention that any existing SPAM mitigation code must be migrated to the plugin

**Comment by SiddheshKukade** [comment]
> 

Ok Sir will work on that 👍🏻

**Review by palisadoes — APPROVED** [review]

**Pull request merged**